pkg/utils/kubernetes/crd.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var (
 	// WaitTimeout specifies the total time to wait for CRDs to become ready or to be deleted. Exposed for testing.
-	WaitTimeout = 15 * time.Second
+	WaitTimeout = 2 * time.Minute
```

Comment:
Add a comment here why this is "so high"

Reply:
Added comment with https://github.com/gardener/gardener/pull/11503/commits/cbff515e208163a0716e7b3d2019fa6694870857.
PTAL.